wealthy-senior-execs/config/_breakpoints.scss

```diff
@@ -0,0 +1,8 @@
+$mobile-upper: 599px;
```

Comment:
Have spoke to Ethan about this and we don't think we need these as the breakpoints aren't specific to WSE, we will use the main site ones